source/guides/maintenance/developer-workstation.rst

```diff
@@ -17,67 +17,213 @@ other Linux\* distributions.
+
+      .. list-table:: 
+         :widths: 50, 50
+         :header-rows: 1
+
+         * - Function
+           - Bundle
+
+         * - All packages required to build the games bundle.
+           - `games-dev <https://clearlinux.org/software/bundle/games-dev/>`_
+
+         * - Godot game engine editor.
+           - `Godot <https://clearlinux.org/software/flathub/godot/>`_
+
+         * - Real-time game engine (Flatpak).
+           - `Unity Hub <https://clearlinux.org/software/flathub/unity-hub/>`_
+   
+         * - Classic point and click adventure game engine and (Flatpak).
+           - `adventure editor <https://clearlinux.org/software/flathub/adventure-editor/>`_
+
+   .. tab:: Kernel Developer
+
+      .. list-table:: 
+         :widths: 50, 50
+         :header-rows: 1
+
+         * - Function
+           - Bundle
+
+         * - Installs kernel, initrd, kernel config, system map; creates a bootloader entry.
+           - `kernel-install <https://clearlinux.org/software/bundle/kernel-install/>`_
+
+         * - Support module for building/loading via Dynamic Kernel Module System (DKMS) in LTS kernel.
+           - `kernel-lts-dkms <https://clearlinux.org/software/bundle/kernel-lts-dkms/>`_
+
+         * - Support module for building/loading via Dynamic Kernel Module System (DKMS) in native kernel.
+           - `kernel-native-dkms <https://clearlinux.org/software/bundle/kernel-native-dkms/>`_
+
+         * - Support module for building/loading via Dynamic Kernel Module System (DKMS) in AWS kernel.
+           - `kernel-aws-dkms <https://clearlinux.org/software/bundle/kernel-aws-dkms/>`_
+
+         * - Run the Kernel-based Virtual Machine (KVM) with |CL| as a guest under KVM.
+           - `kernel-kvm <https://clearlinux.org/software/bundle/kernel-kvm/>`_
+      
+         * - Linux Test Project.
+           - `ltp <https://clearlinux.org/software/bundle/ltp/>`_
+
+   .. tab:: Maker Developer
+
+      .. list-table:: 
+         :widths: 50, 50
+         :header-rows: 1
+
+         * - Function
+           - Bundle
+
+         * - Basic tools for makers and experimenters.
+           - `maker-basic <https://clearlinux.org/software/bundle/maker-basic/>`_
+
+         * - GIS/Mapping tools for makers.
+           - `maker-gis <https://clearlinux.org/software/bundle/maker-gis/>`_
+
+         * - Electronic Design Tool
+           - `Fritzing <https://clearlinux.org/software/flathub/fritzing>`_
+
+         * -  Open-source electronics prototyping platform.
+           - `arduino-ide <https://clearlinux.org/software/flathub/arduino-ide/>`_
+
+   .. tab:: Embedded Systems Developer
+
+      .. list-table:: 
+         :widths: 50, 50
+         :header-rows: 1
+
+         * - Function
+           - Bundle
+
+         * - Run sandboxed desktop application bundles, or build flatpak packages.
+           - `Flatpak <https://clearlinux.org/software/bundle/flatpak/>`_
+
+         * - NetworkManager plugins and desktop integration.
```

Comment:
I'm not sure where to suggest this goes. Pretty specialized use case, I suspect.